update-requirements

```diff
@@ -72,6 +72,9 @@ def get_requirement(version):
 
         return ['>= {}'.format(min_version), '< {}'.format(max_version)]
 
+    if version.startswith('>=') or version.startswith('<'):
```

Comment:
why not also allow plain `>`? or because `1.0.0-1 > 1.0.0`?

Reply:
I think `>` is weird in package requirements. Exactly because of that example. IMHO you should always use `>= fixed.version` rather than `> broken.version`.